### DISCUSSION TITLE
Mobile aim line customization concept

### DISCUSSION THREAD

**Discussion by ghost** [HasangerGames/suroi#375] [Ideas]
![image](https://github.com/user-attachments/assets/529a63aa-2512-489a-ae6e-d572a760439d)
There's a crosshair tab on mobile but changing it does nothing because you can't use crosshairs on mobile. Here's an idea for a replacement, customizable aim lines.

**Comment by Crimson-Haze** [comment]
I like that idea and don't even use mobile.
